game/CMakeLists.txt

```diff
@@ -12,6 +12,21 @@ if(WIN32)
 	# 1.0 1.0.1 1.0+ 1.0.1+ 1.0-2-g123abcd 1.0.1-5-g123abcd
 	# We use the 2-3 digits of the version as MAJOR.MINOR.PATCH
 	# and the git patch number as TWEAK
+	set(SUBSYSTEM_WIN32 WIN32)
+	if (DEFINED CMAKE_SOURCE_DIR)
+		if(DEFINED ENV{ORIGINAL_TEMP})
+			set (TMP_LOCATION $ENV{ORIGINAL_TEMP})
+		elseif(DEFINED ENV{TEMP})
+			set (TMP_LOCATION $ENV{TEMP})
+		else()
+			message(WARNING "Temp folder not found. CPack won't be able to create a proper installer.")
+		endif()
+		file(WRITE "${TMP_LOCATION}/source.txt" "${CMAKE_SOURCE_DIR}")
+		file(WRITE "${TMP_LOCATION}/mingwroot.txt" "${CPACK_MINGW_SYSTEM_PREFIX}")
+	endif()
```

Comment:
cpack by default builds/installs into your defined build and install prefix. No need to redefine that logic...

Reply:
The problem is actually that CPACK doesn't know the source directory where the top-level `CMakeLists.txt` is. This is chiefly a hack for that.